core/src/main/java/de/bwaldvogel/mongo/backend/EmptyCursor.java

```diff
@@ -33,4 +34,9 @@ public boolean isEmpty() {
     public String toString() {
         return getClass().getSimpleName() + "()";
     }
+
+    @Override
+    public OplogPosition getPosition() {
+        return null;
```

Comment:
It this supposed to be ever invoked?
If not, wouldn’t it be cleaner to throw an `UnsupportedOperationException`?

OTH, if it’s never supposed to be invoked, to me this fact suggests that the `TailableCursor` refactoring might not be a good idea after all.